deployments/prod/environment.py

```diff
@@ -1801,6 +1827,7 @@ def env() -> Mapping[str, Optional[str]]:
             for atlas, catalog, sources in [
                 ('hca', 'dcp50', dcp50_sources),
                 ('hca', 'dcp51', dcp51_sources),
+                ('hca', 'dcp52', dcp52_sources),
```

Comment:
We can't add 52 before releasing 51 so we'll have to remove this one from the promo and put it on the back-burner.